tests/ttnn/multichip_unit_tests/test_multidevice_TG.py

```diff
@@ -0,0 +1,196 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+import typing
+import pytest
+import ttnn
+import tempfile
+from loguru import logger
+from tests.ttnn.utils_for_testing import assert_with_pcc
+
+
+from ttnn import (
+    ShardTensorToMesh,
+    ReplicateTensorToMesh,
+    ConcatMeshToTensor,
+    ListMeshToTensor,
+    TensorToMesh,
+    MeshToTensor,
+)
+
+
+@pytest.mark.parametrize(
+    "device_mesh",
+    [
+        32,
+    ],
+    indirect=True,
+)
+def test_galaxy_matmul_1d_fracture(device_mesh):
+    from tests.tt_eager.python_api_testing.sweep_tests.comparison_funcs import comp_pcc
+
+    act_pt = torch.randn(1, 1, 32, 8192)
+    weights_pt = torch.randn(1, 1, 8192, 32768)
+    act = ttnn.from_torch(
+        act_pt,
+        dtype=ttnn.bfloat16,
+        layout=ttnn.TILE_LAYOUT,
+        device=device_mesh,
+        mesh_mapper=ReplicateTensorToMesh(device_mesh),
+    )
+    weights = ttnn.from_torch(
+        weights_pt,
+        dtype=ttnn.bfloat16,
+        layout=ttnn.TILE_LAYOUT,
+        device=device_mesh,
+        mesh_mapper=ShardTensorToMesh(device_mesh, dim=3),
+    )
+
+    gt = act_pt @ weights_pt
+
+    act = ttnn.to_device(act, device_mesh)
```

Comment:
why do you need these two API calls? the `device=device_mesh` in the `ttnn.from_torch` should've moved it onto device-mesh?